src-test/com/zone5cloud/http/core/BaseTest.java

```diff
@@ -74,8 +76,9 @@ public String getBaseEndpoint() {
 	
 	@Before
 	public void init() {
+    	Z5HttpClient.get().setClientConfig(clientConfig);
 		Z5HttpClient.get().setHostname(TEST_SERVER);
-		Z5HttpClient.get().setClientIDAndSecret(TEST_CLIENT_ID, TEST_CLIENT_SECRET);
+		Z5HttpClient.get().setClientIDAndSecret(clientConfig.getClientID(), clientConfig.getClientSecret());
 		Z5HttpClient.get().setDebug(true);
```

Comment:
setClientConfig is setting clientID and clientSecret so you don't need to call both that and setClientIDAndSecret(). See my comment on setClientConfig() also setting host, in which case you could also not call setHostname.